src/core/model_config.proto

```diff
@@ -1290,4 +1290,13 @@ message ModelConfig
   //@@     model.
   //@@
   repeated ModelWarmup model_warmup = 16;
+
+  //@@  .. cpp:var:: string custom_library_path
+  //@@
+  //@@     Optional path of the custom library that contains custom
+  //@@     operations that are used in this specific model.
+  //@@     Necssary only for ONNXRuntime models. Please use LD_PRELOAD for other
+  //@@     types of models.
+  //@@
+  string custom_library_path = 18;
```

Comment:
This needs to be a repeated string in general. Does ONNXRT API allow loading multiple op libraries?

Reply:
The API docs don't explicitly mention if multiple op libraries can be loaded in the same session. I have asked regarding the same. Do we still want it to be a repeated string if multiple op libraries are not supported?

Reply:
Yes, because in the future it may be supported in onnxrt or some other framework. I should have also said that instead of having the repeated directly in the top-level ModelConfig you should have a ModelOperations message. For now that message just has the op library member but we may have other things in the future.

Reply:
Sure, I agree that having a ModelOperations message would be neater as well. I'll make this change

Reply:
Their reply was: https://github.com/microsoft/onnxruntime/issues/4192#issuecomment-643447705
I have restructure the message and made op_library_filename a repeated field.